Includes/font.cpp

```diff
@@ -2,14 +2,20 @@
 
 Font::Font(const char* path) {
   font = TTF_OpenFont(path, 24);
+  outline_font = TTF_OpenFont(path, 24);
```

Comment:
A bigger font size could be nice for readability.

Reply:
Maybe bolder too?

Reply:
With a lower alpha of the outline, I think the current font size and weight actually works.